public/app/core/internationalization/index.tsx

```diff
@@ -69,13 +75,6 @@ export function changeLanguage(locale: string) {
   return i18n.changeLanguage(validLocale);
 }
 
-type I18NextTransType = typeof I18NextTrans;
-type I18NextTransProps = Parameters<I18NextTransType>[0];
-
-interface TransProps extends I18NextTransProps {
-  i18nKey: string;
-}
-
 export const Trans = (props: TransProps): ReactElement => {
   return <I18NextTrans shouldUnescape ns={NAMESPACES} {...props} />;
 };
```

Comment:
Looks like we don't have automatic plugin namespacing for `<Trans />` here? 

Fine for now I think just something to keep in mind/make an issue for.

Reply:
Good catch, maybe I should fix it right away @joshhunt , WDYT?

Reply:
You can pass in an alternate `t` function that's pre-bound to the plugin namespace like

```ts
const t = useTranslate();
return <Trans ... t={t} />
```

but I think just overriding `ns` prop away from the default namespaces (grafana and enterprise)  would do the trick just as well.

https://react.i18next.com/latest/trans-component#trans-props